echoregions/regions2d/regions2d.py

```diff
@@ -501,3 +501,226 @@ def mask(
             M = M.rename(mask_var)
 
         return M
+
+    def transect_mask(
+        self,
+        da_Sv: DataArray,
+        transect_dict: dict = {
+            "start": "ST",
+            "break": "BT",
+            "resume": "RT",
+            "end": "ET",
+        },
+        file_names: List[str] = None,
+    ) -> DataArray:
+        """Mask data from Data Array containing Sv data based off of a Regions2D object
+        and its transect_values.
+
+        Parameters
+        ----------
+        da_Sv : Data Array
+            DataArray of shape (ping_time, depth) containing Sv data.
+        transect_dict : dict
+            Dictionary for transect values. Values must be unique and must be of same length.
```

Comment:
why same length?

Reply:
Mostly so I could easily get the transect type without the number simply by using the length of the transect type. I changed it so that it now just extracts the transect type from the dictionary instead of using lengths.